examples/src/models/embeddings/googlepalm.ts

```diff
@@ -1,16 +1,14 @@
 import { GooglePaLMEmbeddings } from "langchain/embeddings/googlepalm";
 
-export const run = async () => {
-  const model = new GooglePaLMEmbeddings({
-    apiKey: "<YOUR API KEY>", // or set it in environment variable as `GOOGLE_PALM_API_KEY`
-    modelName: "models/embedding-gecko-001", // OPTIONAL
-  });
-  /* Embed queries */
-  const res = await model.embedQuery(
-    "What would be a good company name for a company that makes colorful socks?"
-  );
-  console.log({ res });
-  /* Embed documents */
-  const documentRes = await model.embedDocuments(["Hello world", "Bye bye"]);
-  console.log({ documentRes });
-};
+const model = new GooglePaLMEmbeddings({
```

Comment:
I'm starting to dislike this commit cause it's messing up with other embeddings 😬

Reply:
to give more clarifications - with new  `top-level async-await` ability this is all okay, but if someone is using `import { run } ...` from this module then it would fail for them

Reply:
Fair, but these are just displayed in the docs though and aren't really meant to be imported. We're moving towards using top-level `async/await` for all examples and just thought I'd take the opportunity to change it as I saw it

Reply:
They also enforce that examples in the docs at least pass the TS compiler/linter/formatter